test/stub-node-only.js

```diff
@@ -1254,27 +1312,26 @@ describe("tests that only work against stub server", function () {
             from: "0x00bae5113ee9f252cceb0001205b88fad175461a",
             to: "0x482c57abdce592b39434e3f619ffc3db62ab6d01",
             gas: "0xfffffffff",
-            params: []
+            params: [],
           };
           server.addResponder(function (jso) { if (jso.method === "eth_estimateGas") return expectedResults; });
-          rpc.callOrSendTransaction(payload, function (resultOrError) {
-            assert.strictEqual(resultOrError, expectedResults);
+          rpc.callOrSendTransaction(payload, function (err, result) {
+            assert.isNull(err);
+            assert.strictEqual(result, expectedResults);
             done();
           });
         });
 
         it("can subscribe to new blocks", function (done) {
-          var called = false;
-          rpc.getBlockStream().subscribeToOnBlockAdded(function (block) { done(); });
+          rpc.getBlockStream().subscribeToOnBlockAdded(function (/*block*/) { done(); });
```

Comment:
Can `block` be removed here (and below)?

Reply:
It's intentionally commented out to make it clear (1) what the function signature is, and (2) that the parameter is left unused intentionally.